Directory.Build.props

```diff
@@ -1,13 +1,15 @@
 <Project>
   <PropertyGroup>
-		<Company>Kentico</Company>
-		<Authors>Eric Dugre</Authors>
-    <VersionPrefix>4.0.0</VersionPrefix>
-		<Product>Xperience by Kentico</Product>
-		<LangVersion>12.0</LangVersion>
-		<ImplicitUsings>enable</ImplicitUsings>
-		<Nullable>enable</Nullable>
-	</PropertyGroup>
+    <Company>Kentico</Company>
+	<Authors>Eric Dugre</Authors>
+    <VersionPrefix>4.1.0</VersionPrefix>
```

Comment:
You are making significant changes in public members which is considered to be a breaking change. We should bump to new major version 5.0.0. Also please fix the indentation.